### PR TITLE
Proceedings reference

### LINKED ISSUE #11: [URGENT] Standardize Proceedings Citation.
At some point this year I had to cite the ISQCMC Berlin proceedings as a whole, for a book chapter. This is what I ended up using in my case, in APA Format:
Hamido, O. C. and Itaboraı́, P. V. (eds.) (2023b). Proceedings of the 2nd Inter-
national Symposium on Quantum Computing and Musical Creativity (Zen-
odo), https://isqcmc.github.io/Proceedings, [Conference proceedings].

I've now exchanged some emails with another ISQCMC author, and they refered to the proceedings, while citing their paper like so:

Authors, Title. arxivDOI. In: Proceedings of the 2nd International Symposium on Quantum Computing and Musical Creativity 2023, K. Jansen (ed) (2023), PDF/DOI: 10.5281/zenodo.XXXXX

This means we need to update this information urgently, so we make sure the citations are correct and uniform - and easily accessible.

**TODO:**
- [x] edit the about page of zenodo
- [x] add reference of entire proceedings

### PR DESCRIPTION
**Pull request by omarcostahamido**
- this fixes #11

### CODE REVIEW

**Review by Itaborala — APPROVED** [review]

**Review by Itaborala — CHANGES_REQUESTED** [review]
Two additional ideas:

- [ ] Should we make a link to the [Zenodo Community](https://zenodo.org/communities/isqcmc_proceedings/) on the top of the page as well?
- [ ] One idea just occured: we make a github release of this repo containing the `Proceedings.md` file. Then we publish this release on Zenodo to obtain a DOI for the proceedings as a whole ....?

**Comment by omarcostahamido** [comment]
@Itaborala this PR got lost in the woods

> - [ ] Should we make a link to the [Zenodo Community](https://zenodo.org/communities/isqcmc_proceedings/) on the top of the page as well?

Sure

> - [ ] One idea just occured: we make a github release of this repo containing the `Proceedings.md` file. Then we publish this release on Zenodo to obtain a DOI for the proceedings as a whole ....? 

It's a wonky idea but it may work... it is weird because one would cite as proceedings a document where the paper is not readable, but only linked to. I think this project looks much more close to what NIME does. 
edit: actually, just looking at an entry at nime.org it look like they make the references in such a way as if they were part of a book that doesn't seem to exist, they even say in the bib citation the number of pages... interesting. 
Thoughts?

**Comment by omarcostahamido** [comment]
@Itaborala ready for you to review

**Comment by omarcostahamido** [comment]
let me know what you think of this last change as well https://github.com/isqcmc/isqcmc.github.io/pull/12/commits/30e2d69d5230ffea33c045fcd07c9c8057a1f0b2

**Comment by Itaborala** [comment]
> It's a wonky idea but it may work... it is weird because one would cite as proceedings a document where the paper is not readable, but only linked to. I think this project looks much more close to what NIME does. edit: actually, just looking at an entry at nime.org it look like they make the references in such a way as if they were part of a book that doesn't seem to exist, they even say in the bib citation the number of pages... interesting. Thoughts?

Also just looked in the NIME proceedings, and they apparently have a separate Zenodo Community called _Archive_ of the NIME proceedings, where they publish [zip files](https://zenodo.org/records/4016363) with all the papers. Maybe the page ordering comes from there?

I also remember receiving a usb stick as a gift during NIME 2019, which contained all the papers - if my memory is not tricking me - inside a single PDF file.

All in all, I'm not a super fan of using page number in this situation, but I understand they did it also to comply with all citation requirements out there.

An updated suggestion would be to put everything in a zip file and call that the compiled proceedings of ISQCMC Berlin, with a DOI...

**Comment by omarcostahamido** [comment]
do we need an outsider's opinion to solve this one?

File: Proceedings.md [hunk 1]
```diff
@@ -30,3 +34,8 @@
   
 - Yáñez, M. A. (2023). _Developing a Quantum Step-sequencer_. **PDF/DOI: [10.5281/zenodo.10206500](https://doi.org/10.5281/zenodo.10206500)** <citebutton-component indexkey="ISQCMC_yanez"></citebutton-component>
 
+### 2021 - 1st ISQCMC - Paper Proceedings
+
+Paper Proceedings for this edition are not available. However, please note that several papers from this edition have evolved into book chapters in:
```
Comment (Itaborala) [inline Proceedings.md:39]: It does make sense to specify it here, I agree. 

Do you think we should also mention the new book and link it to 2023?

**Comment by Itaborala** [comment]
> do we need an outsider's opinion to solve this one?

Would be good. I can ask some people here

**Comment by omarcostahamido** [comment]
@medusamedusa
i know this is not directly about isqcmc2025, but maybe you could give us some suggestion as well?
😇

**Comment by medusamedusa** [comment]
Sorry, I do not understand :)

Are we going to work on Wordpress, and so I ca share the password, or on GitHub, and so I will ask you how to access and edit it?

BTW, can you please put the logo information before the web design? Thanks :))
Maria

**Comment by medusamedusa** [comment]
I prefer to have a DOI for each article separately.

> * [ ]  One idea just occured: we make a github release of this repo containing the `Proceedings.md` file. Then we publish this release on Zenodo to obtain a DOI for the proceedings as a whole ....?

**Comment by medusamedusa** [comment]
> @medusamedusa i know this is not directly about isqcmc2025, but maybe you could give us some suggestion as well? 😇

I am now able to read and reply here :)

**Comment by omarcostahamido** [comment]
@medusamedusa 

> Sorry, I do not understand :)

we are discussing how to create a reference to the entire proceedings of 2023, when we have only published the papers individually on zenodo. Have you been in this situation before? which of the solutions discussed above make sense, or none? It's unrelated to 2025, sorry. but I just thought of asking for an opinion.

> Are we going to work on Wordpress, and so I ca share the password, or on GitHub, and so I will ask you how to access and edit it?

No wordpress please! Just on github. You already have access to https://github.com/isqcmc/isqcmc2025 which is the repo for the 2025 website. you can edit all that you want and propose those changes via Pull request. And you can also open as many issues there as you want to request help to make changes...

> BTW, can you please put the logo information before the web design? Thanks :))

such as this one :D

> I prefer to have a DOI for each article separately.

each article already has its own individual DOI (since late 2023 or early 2024). The question here now is only how to refer to the entire proceedings.

> I am now able to read and reply here :)

Perfect! Yes, it is good to have some context for the messages haha :)

**Review by Itaborala — APPROVED** [review]

**Pull request merged**